src/i18n/en/translation.json

```diff
@@ -311,6 +311,7 @@
       "successModalLabel": "Your answers have been submitted. Please return to the Admin App to complete another activity or view results.",
       "successModalPrimaryAction": "Return to Admin App",
       "successModalSecondaryAction": "Dismiss"
-    }
+    },
+    "charactersCount": "({{numCharacters}}/{{maxCharacters}}) characters"
```

Comment:
Overall this is fine but I noticed this includes the parenthesis as part of the text, which is unexpected

| Figma | Web app |
|-|-|
| ![CleanShot 2024-07-30 at 16 37 10](https://github.com/user-attachments/assets/25353c67-8627-4028-8399-f401a8ef1995)  |  ![CleanShot 2024-07-30 at 16 37 33](https://github.com/user-attachments/assets/25f7d646-1cf0-4c06-86ea-2ea68ae04c98)  |


Hoping you can fix this in both languages and should be good to go